src/commands/move.ts

```diff
@@ -90,16 +90,25 @@ export class Move extends Command {
 
                 MessageAttachments.push(new MessageAttachment(content, messageMessageAttachment.name ?? undefined));
             }
-
-            let options: WebhookMessageOptions & { split?: false } = {
-                username: `${message.author.username}`,
-                avatarURL: message.author.avatarURL() ?? undefined,
-                files: MessageAttachments,
-                embeds: message.embeds,
-                disableMentions: "all",
-            };
-
-            await webhook.send(message.content, options);
+            if(message.content.length > 0){
+                await webhook.send({
+                    content: message.content,
+                    username: message.author.username,
+                    avatarURL: message.author.avatarURL() ?? undefined,
+                    files: MessageAttachments,
+                    embeds: message.embeds,
+                    allowedMentions: {parse: []},
+                });
+            }else{
```

Comment:
There should be spaces around `else` and after `if` and before the `{` brace.

Reply:
done